src/components/memorygame/MemoryGame/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from "react";
+import { Col, Row } from "antd";
+import Confetti from "react-dom-confetti";
+import { Redirect } from "react-router-dom";
+import { useTranslation } from "react-i18next";
+import MemoryGameFactList from "../MemoryGameFactList/index";
+import "./index.css";
+import confettiConfig from "../confettiConfig";
+import MemoryGameCards from "../MemoryGameCards";
+
+const MemoryGame = () => {
+  const [t] = useTranslation();
+  const factsTitle = t("memoryGame.facts");
+  const gameTitle = t("memoryGame.title");
+  const [completed, setCompleted] = useState(false);
+  const [redirect] = useState();
```

Comment:
why is there no set function or default value for this state?

Reply:
Redirect is obsolete. Code is not ready for review yet. I was waiting for other PR